content/en/docs/ibm/using-icr.md

```diff
@@ -0,0 +1,80 @@
++++
+title = "Using IBM Cloud Container Registry (ICR)"
+description = "Instructions for using private images from ICR"
+weight = 15
++++
+
+This guide describe steps for use cases involving private container images by using IBM Cloud Container Registry (ICR).
+
+## Prerequisites
+
+* Install and configure the [IBM Cloud CLI](https://cloud.ibm.com/docs/cli?topic=cli-getting-started).
+* Install the CLI plug-in for the IBM Cloud Container Registry by running the command `ibmcloud plugin install container-registry`.
+* Create a namespace in ICR with command `ibmcloud cr namespace-add <my_namespace>`, replace `<my_namespace>` with your preferred name.
+
+**Note**: the [ICR namespace](https://cloud.ibm.com/docs/Registry?topic=Registry-getting-started#gs_registry_namespace_add) is different from the Kubeflow Profile namespace. The ICR namespace is used to group container images stored in ICR while a [Kubeflow Profile](/docs/components/multi-tenancy/overview/) namespace is a grouping of all Kubernetes clusters onwed by a user.
+
+## Image pull secret
+
+An image pull secret is required to to pull container images from IBM Cloud Container Registry. You can use the default image pull secret set up by the cluster or use your account's IAM API key.
+
+### Use default image pull secret
```

Comment:
Small change:

- Suggest to change it to:

```
### Using a default image pull secret
```

("Using" since this is a guide, it makes a better headline.)